packages/fxa-settings/src/models/Account.ts

```diff
@@ -897,6 +897,13 @@ export class Account implements AccountData {
     );
   }
 
+  async saveRecoveryCodeHint(hint: string) {
```

Comment:
If rebased on main, you should be able to use `Account.ts`'s existing `updateRecoveryKeyHint` method (already linked to existing auth client method).